tests/test_fireworks.py

```diff
@@ -0,0 +1,110 @@
+"""Tests for Rago package using Fireworks AI for gen and aug."""
```

Comment:
I made some changes in the tests .. please add your tests into the new structure. 
you can add your tests to 

https://github.com/osl-incubator/rago/blob/main/tests/test_generation.py

and 

https://github.com/osl-incubator/rago/blob/main/tests/test_augmentation.py


let me know if you have any questions